release-notes/8.0/8.0.1/api-diff/Microsoft.NETCore.App/8.0.1_System.Runtime.Intrinsics.md

```diff
@@ -0,0 +1,441 @@
+# System.Runtime.Intrinsics
+
+``` diff
+ namespace System.Runtime.Intrinsics {
```

Comment:
Shared intrinsics LGTM. This is Vector512<T> and some additional APIs exposed for consistency.